tests/entrypoints/openai/tool_use/conftest.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from huggingface_hub import snapshot_download
+
+from ....utils import RemoteOpenAIServer
+from .util import ARGS, CONFIGS, ServerConfig
+
+
+# for each server config, download the model and return the config
+@pytest.fixture(scope="module", params=CONFIGS.keys())
+def server_config(request):
+    config = CONFIGS[request.param]
+    # download model and tokenizer using transformers
+    snapshot_download(config["model"])
```

Comment:
Add the startup parameter --ignore-patterns="consolidated.safetensors" for mistralai/Mistral-7B-Instruct-v0.3 to avoid downloading this file. Then you can use the `from_pretrained method` to download. This solution seems to not require modifying the timeout.